pgbouncer/templates/deployment-pgbouncer.yaml

```diff
@@ -52,17 +40,12 @@ spec:
                 topologyKey: kubernetes.io/hostname
                 labelSelector:
                   matchLabels:
-                    {{ template "pgbouncer.appLabel" . }}
-                    release: {{ .Release.Name }}
+                    {{- include "pgbouncer.selectorLabels" . | nindent 20 }}
         {{- end }}
         {{- if .Values.nodeAffinity }}
         nodeAffinity:
           {{- toYaml .Values.nodeAffinity | nindent 10 }}
         {{- end }}
-      {{- if .Values.nodeSelector }}
-      nodeSelector:
```

Comment:
why did you remove nodeSelector?  this could break existing deployments

Reply:
Fixed